ballerina/tests/bulk_csv_abort.bal

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/log;
+import ballerina/test;
+
+@test:Config {
+    enable: true
+}
+function abortAndDeleteJob() returns error? {
+    log:printInfo("baseClient -> deleteCsv");
+    //create job
+    BulkCreatePayload payload = {
+        'object : "Contact",
+        contentType : "CSV",
+        operation : "delete",
+        lineEnding : "LF"
+    };
+    BulkJob abortJob = check baseClient->createJob(payload, INGEST);
+
+    log:printInfo("baseClient -> abortJob");
+    error? abortJobError = baseClient->abortJob(abortJob.id, INGEST);
+    if abortJobError is error {
+        test:assertFail(msg = abortJobError.message());
+    }
+    log:printInfo("baseClient -> deleteJob");
+    error? deleteJobError = baseClient->deleteJob(abortJob.id, INGEST);
+    if deleteJobError is error {
+        test:assertFail(msg = deleteJobError.message());
+    }
+
+}
```

Comment:
```suggestion
}

```

Reply:
Updated in [update code according to comments](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/311/commits/cd6ae536defa51bd247b84742f1c480c2cf80ab1)